budcamp/Homework2/Homework2/Point.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework2
+{
+    struct Point
+    {
+        public double X { get; set; }
+        public double Y { get; set; }
+
+        public Point()
+        {
+            X = 0;
+            Y = 0;
+        }
+
+        public Point(int x, int y)
+        {
+            this.X = x;
+            this.Y = y;
+        }
+
+        public bool TryParsePoint(string input, out Point point)
+        {
+            point = default(Point);
+
+            try
+            {
+                string[] splitedString = input.Split(',');
+                splitedString[0].Trim();
+                splitedString[1].Trim();
```

Comment:
string is immutable, so Trim() method doesn't change the array value - it's returning a trimmed value.
if you wanted to trim it you should have to do it like `splitedString[0] = splitedString[0].Trim()` 
But I'd change your split method to `Split(',', StringSplitOptions.TrimEntries)`

Reply:
done